### DISCUSSION TITLE
Got manuals for SSD202D

### DISCUSSION THREAD

**Discussion by ltrsunny** [linux-chenxing/linux-chenxing.org#62] [General]
which provides more information than current datasheet.
Found from a weird BBS
[SSD202D.pdf](https://github.com/linux-chenxing/linux-chenxing.org/files/8706238/SSD202D.pdf)
.

**Comment by fifteenhex** [comment]
Thanks! I will check if it's different to the others we have.

**Reply by ltrsunny** [reply]
> Thanks! I will check if it's different to the others we have.

Dongshan pi released more https://cowtransfer.com/s/0756a2af637745
Maybe you already have.

**Reply by fifteenhex** [reply]
I think I have all of those. Thanks for the info though.
